### ISSUE TITLE
specs2 with scalacheck: show stack trace

### ISSUE THREAD

**Issue by etaty** [etorreborre/specs2#490]
I would expect to see the stack trace when a prop is falsified

```scala
import org.specs2.ScalaCheck
import org.specs2.mutable.Specification
import org.specs2.scalacheck.Parameters

class ScalaCheckSpec extends Specification with ScalaCheck {
  implicit val params = Parameters(minTestsOk = 20).verbose

  "this is a specific property" >> prop { (a: Int, b: Int) =>
    1 === 10
  }.set(minTestsOk = 20, workers = 1).noShrink.verbose
}
```

```
[error] x this is a specific property
[error]  Falsified after 0 passed tests.
[error]  > ARG_0: 0
[error]  > ARG_0_ORIGINAL: 513834775
[error]  > ARG_1: -2147483648
[error]  > '1' is not equal to '10' (ScalaCheckSpec.scala:5)
```
I would expect the error to point to the line of the failed Expectation ( `1 === 10` is on line 9)
instead of the line 5 `class ScalaCheckSpec  ... `
And a stack trace would be even better.

**Comment by etorreborre** [comment]
I can't really check that at the moment but can you try to pass the `failtrace` argument? This is how you get stacktraces on failures.

**Comment by etaty** [comment]
`test-only ScalaCheckSpec -- failtrace` has no effect with an `Expectation`.
It works with `???` and give the correct line and stack.


I am thinking this line is creating the problem?
https://github.com/etorreborre/specs2/blob/5a556adb38f065c3c165208ca89f5fc8d682d71b/scalacheck/src/main/scala/org/specs2/scalacheck/AsResultProp.scala#L23

**Comment by etorreborre** [comment]
I think you are right. I am not sure how this can be improved but I'll have a go at it.

**Comment by etorreborre** [comment]
This is fixed now in `3.8.4-20160711064123-77be371`. Now if you use `failtrace`, you should see a proper stacktrace for your failure.

**Comment by etaty** [comment]
Thanks 👍
